rhp/v4/rpc_test.go

```diff
@@ -543,16 +543,33 @@ func TestRPCRefresh(t *testing.T) {
 
 	t.Run("valid refresh", func(t *testing.T) {
 		revision := formContractUploadSector(t, types.Siacoins(100), types.Siacoins(200), types.Siacoins(25))
+
+		// compute the min allowance the host will accept
```

Comment:
There shouldn't be a need to change this test right? Forming a contract with 100/200 SC then refreshing it to add 10/20 should be a totally valid ratio.